sqlmesh/core/console.py

```diff
@@ -483,6 +458,75 @@ def _get_ignored_tree(
             )
         return ignored
 
+    def _show_summary_tree_for(
+        self,
+        context_diff: ContextDiff,
+        header: str,
+        filter: t.Callable,
```

Comment:
`filter` is an existing Python keyword (or rather builtin function). I suggest coming up with a different name.

Reply:
If it's a predicate than I think expanding the type signature makes sense: `t.Callable[[Snapshot], bool]`.